src/SourceGenerators/Uno.UI.SourceGenerators/NativeCtor/NativeCtorsGenerator.cs

```diff
@@ -51,7 +51,7 @@ private class SerializationMethodsGenerator : SymbolVisitor
 // This file has been generated by Uno.UI (NativeCtorsGenerator)
 // *************************************************************
 // </auto-generated>
-#pragma warning disable 1591 // Ignore missing XML comment warnings
```

Comment:
This warning is generally required for applications that do have XML comments validation enabled (including parameters). Did you get a warning about the ignore inside of Uno?

Reply:
My initial reasoning was that generated constructors already have doc comments. But just revised that and found that not all of them. The parameterless one doesn't have doc comments, so this line is needed. Closing out.